lang/types/type_test.go

```diff
@@ -1474,4 +1474,5 @@ func TestComplexCmp0(t *testing.T) {
 
 func TestTypeOf0(t *testing.T) {
 	// TODO: implement testing of the TypeOf function
```

Comment:
commit message: "should (almost) bijective" missing a word? `be`??

Reply:
I have no what you're talking about :grin: